PRAGMA foreign_keys=OFF;
BEGIN TRANSACTION;
CREATE TABLE IF NOT EXISTS "django_migrations" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "app" varchar(255) NOT NULL, "name" varchar(255) NOT NULL, "applied" datetime NOT NULL);
INSERT INTO django_migrations VALUES(1,'contenttypes','0001_initial','2025-02-08 19:06:51.815025');
INSERT INTO django_migrations VALUES(2,'auth','0001_initial','2025-02-08 19:06:51.836606');
INSERT INTO django_migrations VALUES(3,'admin','0001_initial','2025-02-08 19:06:51.856920');
INSERT INTO django_migrations VALUES(4,'admin','0002_logentry_remove_auto_add','2025-02-08 19:06:51.870059');
INSERT INTO django_migrations VALUES(5,'admin','0003_logentry_add_action_flag_choices','2025-02-08 19:06:51.882776');
INSERT INTO django_migrations VALUES(6,'contenttypes','0002_remove_content_type_name','2025-02-08 19:06:51.897870');
INSERT INTO django_migrations VALUES(7,'auth','0002_alter_permission_name_max_length','2025-02-08 19:06:51.912134');
INSERT INTO django_migrations VALUES(8,'auth','0003_alter_user_email_max_length','2025-02-08 19:06:51.922067');
INSERT INTO django_migrations VALUES(9,'auth','0004_alter_user_username_opts','2025-02-08 19:06:51.931613');
INSERT INTO django_migrations VALUES(10,'auth','0005_alter_user_last_login_null','2025-02-08 19:06:51.944097');
INSERT INTO django_migrations VALUES(11,'auth','0006_require_contenttypes_0002','2025-02-08 19:06:51.950516');
INSERT INTO django_migrations VALUES(12,'auth','0007_alter_validators_add_error_messages','2025-02-08 19:06:51.958016');
INSERT INTO django_migrations VALUES(13,'auth','0008_alter_user_username_max_length','2025-02-08 19:06:51.970231');
INSERT INTO django_migrations VALUES(14,'auth','0009_alter_user_last_name_max_length','2025-02-08 19:06:51.985030');
INSERT INTO django_migrations VALUES(15,'auth','0010_alter_group_name_max_length','2025-02-08 19:06:51.998653');
INSERT INTO django_migrations VALUES(16,'auth','0011_update_proxy_permissions','2025-02-08 19:06:52.010787');
INSERT INTO django_migrations VALUES(17,'auth','0012_alter_user_first_name_max_length','2025-02-08 19:06:52.020162');
INSERT INTO django_migrations VALUES(18,'sessions','0001_initial','2025-02-08 19:06:52.033031');
INSERT INTO django_migrations VALUES(19,'Users','0001_initial','2025-02-08 19:56:56.764793');
INSERT INTO django_migrations VALUES(20,'Users','0002_alter_question_correct_answer','2025-02-08 19:56:56.775974');
INSERT INTO django_migrations VALUES(21,'Users','0003_topic_subtopic_name_alter_question_question_name_and_more','2025-02-09 16:16:08.403955');
INSERT INTO django_migrations VALUES(22,'Users','0004_alter_user_question_unique_together','2025-02-12 23:46:30.307355');
CREATE TABLE IF NOT EXISTS "auth_group_permissions" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "group_id" integer NOT NULL REFERENCES "auth_group" ("id") DEFERRABLE INITIALLY DEFERRED, "permission_id" integer NOT NULL REFERENCES "auth_permission" ("id") DEFERRABLE INITIALLY DEFERRED);
CREATE TABLE IF NOT EXISTS "auth_user_groups" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "user_id" integer NOT NULL REFERENCES "auth_user" ("id") DEFERRABLE INITIALLY DEFERRED, "group_id" integer NOT NULL REFERENCES "auth_group" ("id") DEFERRABLE INITIALLY DEFERRED);
CREATE TABLE IF NOT EXISTS "auth_user_user_permissions" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "user_id" integer NOT NULL REFERENCES "auth_user" ("id") DEFERRABLE INITIALLY DEFERRED, "permission_id" integer NOT NULL REFERENCES "auth_permission" ("id") DEFERRABLE INITIALLY DEFERRED);
CREATE TABLE IF NOT EXISTS "django_admin_log" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "object_id" text NULL, "object_repr" varchar(200) NOT NULL, "action_flag" smallint unsigned NOT NULL CHECK ("action_flag" >= 0), "change_message" text NOT NULL, "content_type_id" integer NULL REFERENCES "django_content_type" ("id") DEFERRABLE INITIALLY DEFERRED, "user_id" integer NOT NULL REFERENCES "auth_user" ("id") DEFERRABLE INITIALLY DEFERRED, "action_time" datetime NOT NULL);
INSERT INTO django_admin_log VALUES(1,'1','Test Topic',1,'[{"added": {}}]',8,1,'2025-02-08 20:05:31.330783');
INSERT INTO django_admin_log VALUES(2,'1','Prasanna Thapa',1,'[{"added": {}}]',9,1,'2025-02-08 20:05:57.014962');
INSERT INTO django_admin_log VALUES(3,'1',replace('\n\n        Why did the chicken cross the road?\n        \n        OPTIONS:\n        To get to the other side\n        He was hungry for some drumsticks - cannibalism is okay in a joke, right?\n        This ','\n',char(10)),1,'[{"added": {}}]',7,1,'2025-02-08 20:07:42.502976');
INSERT INTO django_admin_log VALUES(4,'1','User_Question object (1)',1,'[{"added": {}}]',10,1,'2025-02-08 20:08:04.131156');
INSERT INTO django_admin_log VALUES(5,'3',replace('\n\n        Which of the following amino acids is MOST likely to be found on the surface of a protein in an aqueous solution?\n        \n        OPTIONS:\n        {''1. Valine''}\n        {''2. Leucine''}\n     ','\n',char(10)),3,'',7,1,'2025-02-09 14:56:27.872594');
INSERT INTO django_admin_log VALUES(6,'1','User_Question object (1)',3,'',10,1,'2025-02-09 16:34:12.054945');
INSERT INTO django_admin_log VALUES(7,'1',replace('\n\n        Why did the chicken cross the road?\n        \n        OPTIONS:\n        To get to the other side\n        He was hungry for some drumsticks - cannibalism is okay in a joke, right?\n        This ','\n',char(10)),3,'',7,1,'2025-02-09 16:34:20.035337');
INSERT INTO django_admin_log VALUES(8,'3',replace('\n\n        Which of the following amino acids is MOST likely to be found on the surface of a protein in an aqueous solution?\n        \n        OPTIONS:\n        {''1. Valine''}\n        {''2. Leucine''}\n     ','\n',char(10)),2,'[{"changed": {"fields": ["TOPIC"]}}]',7,1,'2025-02-09 16:34:27.666810');
INSERT INTO django_admin_log VALUES(9,'2',replace('\n\n        Which of the following amino acids is MOST likely to be found on the surface of a protein in an aqueous solution?\n        \n        OPTIONS:\n        {''1. Valine''}\n        {''2. Leucine''}\n     ','\n',char(10)),3,'',7,1,'2025-02-09 16:34:37.111970');
INSERT INTO django_admin_log VALUES(10,'1','Prasanna Thapa',2,'[{"changed": {"fields": ["TOPIC"]}}]',9,1,'2025-02-09 16:34:54.481089');
INSERT INTO django_admin_log VALUES(11,'1','MCAT : Test Topic : test',3,'',8,1,'2025-02-09 16:35:04.902254');
INSERT INTO django_admin_log VALUES(12,'2','User_Question object (2)',3,'',10,1,'2025-02-12 23:44:59.081914');
INSERT INTO django_admin_log VALUES(13,'3','User_Question object (3)',3,'',10,1,'2025-02-12 23:44:59.081962');
INSERT INTO django_admin_log VALUES(14,'4','User_Question object (4)',3,'',10,1,'2025-02-12 23:44:59.081994');
INSERT INTO django_admin_log VALUES(15,'5','User_Question object (5)',3,'',10,1,'2025-02-12 23:44:59.082024');
INSERT INTO django_admin_log VALUES(16,'6','User_Question object (6)',3,'',10,1,'2025-02-12 23:44:59.082053');
INSERT INTO django_admin_log VALUES(17,'7','User_Question object (7)',3,'',10,1,'2025-02-12 23:44:59.082082');
INSERT INTO django_admin_log VALUES(18,'8','User_Question object (8)',3,'',10,1,'2025-02-12 23:44:59.082112');
INSERT INTO django_admin_log VALUES(19,'9','User_Question object (9)',3,'',10,1,'2025-02-12 23:44:59.082141');
INSERT INTO django_admin_log VALUES(20,'10','User_Question object (10)',3,'',10,1,'2025-02-12 23:44:59.082170');
INSERT INTO django_admin_log VALUES(21,'11','User_Question object (11)',3,'',10,1,'2025-02-12 23:44:59.082198');
INSERT INTO django_admin_log VALUES(22,'12','User_Question object (12)',3,'',10,1,'2025-02-12 23:44:59.082226');
INSERT INTO django_admin_log VALUES(23,'13','User_Question object (13)',3,'',10,1,'2025-02-12 23:44:59.082255');
INSERT INTO django_admin_log VALUES(24,'14','User_Question object (14)',3,'',10,1,'2025-02-12 23:44:59.082282');
INSERT INTO django_admin_log VALUES(25,'15','User_Question object (15)',3,'',10,1,'2025-02-12 23:44:59.082311');
INSERT INTO django_admin_log VALUES(26,'16','User_Question object (16)',3,'',10,1,'2025-02-12 23:44:59.082338');
INSERT INTO django_admin_log VALUES(27,'17','User_Question object (17)',3,'',10,1,'2025-02-12 23:44:59.082366');
CREATE TABLE IF NOT EXISTS "django_content_type" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "app_label" varchar(100) NOT NULL, "model" varchar(100) NOT NULL);
INSERT INTO django_content_type VALUES(1,'admin','logentry');
INSERT INTO django_content_type VALUES(2,'auth','permission');
INSERT INTO django_content_type VALUES(3,'auth','group');
INSERT INTO django_content_type VALUES(4,'auth','user');
INSERT INTO django_content_type VALUES(5,'contenttypes','contenttype');
INSERT INTO django_content_type VALUES(6,'sessions','session');
INSERT INTO django_content_type VALUES(7,'Users','question');
INSERT INTO django_content_type VALUES(8,'Users','topic');
INSERT INTO django_content_type VALUES(9,'Users','user');
INSERT INTO django_content_type VALUES(10,'Users','user_question');
CREATE TABLE IF NOT EXISTS "auth_permission" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "content_type_id" integer NOT NULL REFERENCES "django_content_type" ("id") DEFERRABLE INITIALLY DEFERRED, "codename" varchar(100) NOT NULL, "name" varchar(255) NOT NULL);
INSERT INTO auth_permission VALUES(1,1,'add_logentry','Can add log entry');
INSERT INTO auth_permission VALUES(2,1,'change_logentry','Can change log entry');
INSERT INTO auth_permission VALUES(3,1,'delete_logentry','Can delete log entry');
INSERT INTO auth_permission VALUES(4,1,'view_logentry','Can view log entry');
INSERT INTO auth_permission VALUES(5,2,'add_permission','Can add permission');
INSERT INTO auth_permission VALUES(6,2,'change_permission','Can change permission');
INSERT INTO auth_permission VALUES(7,2,'delete_permission','Can delete permission');
INSERT INTO auth_permission VALUES(8,2,'view_permission','Can view permission');
INSERT INTO auth_permission VALUES(9,3,'add_group','Can add group');
INSERT INTO auth_permission VALUES(10,3,'change_group','Can change group');
INSERT INTO auth_permission VALUES(11,3,'delete_group','Can delete group');
INSERT INTO auth_permission VALUES(12,3,'view_group','Can view group');
INSERT INTO auth_permission VALUES(13,4,'add_user','Can add user');
INSERT INTO auth_permission VALUES(14,4,'change_user','Can change user');
INSERT INTO auth_permission VALUES(15,4,'delete_user','Can delete user');
INSERT INTO auth_permission VALUES(16,4,'view_user','Can view user');
INSERT INTO auth_permission VALUES(17,5,'add_contenttype','Can add content type');
INSERT INTO auth_permission VALUES(18,5,'change_contenttype','Can change content type');
INSERT INTO auth_permission VALUES(19,5,'delete_contenttype','Can delete content type');
INSERT INTO auth_permission VALUES(20,5,'view_contenttype','Can view content type');
INSERT INTO auth_permission VALUES(21,6,'add_session','Can add session');
INSERT INTO auth_permission VALUES(22,6,'change_session','Can change session');
INSERT INTO auth_permission VALUES(23,6,'delete_session','Can delete session');
INSERT INTO auth_permission VALUES(24,6,'view_session','Can view session');
INSERT INTO auth_permission VALUES(25,7,'add_question','Can add question');
INSERT INTO auth_permission VALUES(26,7,'change_question','Can change question');
INSERT INTO auth_permission VALUES(27,7,'delete_question','Can delete question');
INSERT INTO auth_permission VALUES(28,7,'view_question','Can view question');
INSERT INTO auth_permission VALUES(29,8,'add_topic','Can add topic');
INSERT INTO auth_permission VALUES(30,8,'change_topic','Can change topic');
INSERT INTO auth_permission VALUES(31,8,'delete_topic','Can delete topic');
INSERT INTO auth_permission VALUES(32,8,'view_topic','Can view topic');
INSERT INTO auth_permission VALUES(33,9,'add_user','Can add user');
INSERT INTO auth_permission VALUES(34,9,'change_user','Can change user');
INSERT INTO auth_permission VALUES(35,9,'delete_user','Can delete user');
INSERT INTO auth_permission VALUES(36,9,'view_user','Can view user');
INSERT INTO auth_permission VALUES(37,10,'add_user_question','Can add user_ question');
INSERT INTO auth_permission VALUES(38,10,'change_user_question','Can change user_ question');
INSERT INTO auth_permission VALUES(39,10,'delete_user_question','Can delete user_ question');
INSERT INTO auth_permission VALUES(40,10,'view_user_question','Can view user_ question');
CREATE TABLE IF NOT EXISTS "auth_group" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "name" varchar(150) NOT NULL UNIQUE);
CREATE TABLE IF NOT EXISTS "auth_user" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "password" varchar(128) NOT NULL, "last_login" datetime NULL, "is_superuser" bool NOT NULL, "username" varchar(150) NOT NULL UNIQUE, "last_name" varchar(150) NOT NULL, "email" varchar(254) NOT NULL, "is_staff" bool NOT NULL, "is_active" bool NOT NULL, "date_joined" datetime NOT NULL, "first_name" varchar(150) NOT NULL);
INSERT INTO auth_user VALUES(1,'pbkdf2_sha256$600000$CKHnkjWXlzycoj3CyWsEV7$RLzAnIhLecb7NwDYx4Om96geNH/vDIhjYK84MHyJYgs=','2025-02-08 20:04:07.611852',1,'royalrivet','','thapapj19@gmail.com',1,1,'2025-02-08 20:03:26.519078','');
CREATE TABLE IF NOT EXISTS "django_session" ("session_key" varchar(40) NOT NULL PRIMARY KEY, "session_data" text NOT NULL, "expire_date" datetime NOT NULL);
INSERT INTO django_session VALUES('wfdy1brwwz9kct1euixoscjr4nwt598d','.eJxVjMsOwiAQRf-FtSE8hoIu3fcbyDADUjUlKe3K-O_apAvd3nPOfYmI21rj1vMSJxYXocXpd0tIjzzvgO8435qkNq_LlOSuyIN2OTbOz-vh_h1U7PVbA3lNCKitygF8OifvgslkkJUKwTEhgitIfgiqsEETYLCWoCBwGYp4fwDo6zhY:1tgr3f:5Uoz6DXpmB8FzvLMpoonc53x7CTk8t_bkSNQA0w_-ZA','2025-02-22 20:04:07.617151');
CREATE TABLE IF NOT EXISTS "Users_user" ("USER_ID" integer NOT NULL PRIMARY KEY, "USER_FIRST_NAME" varchar(100) NOT NULL, "USER_LAST_NAME" varchar(100) NOT NULL, "TOPIC_id" integer NULL REFERENCES "Users_topic" ("TOPIC_ID") DEFERRABLE INITIALLY DEFERRED);
INSERT INTO Users_user VALUES(1,'Prasanna','Thapa',2);
CREATE TABLE IF NOT EXISTS "Users_topic" ("TOPIC_ID" integer NOT NULL PRIMARY KEY, "SUBJECT" varchar(100) NOT NULL, "TOPIC_NAME" varchar(100) NOT NULL, "SUBTOPIC_NAME" varchar(100) NOT NULL);
INSERT INTO Users_topic VALUES(2,'MCAT','Biological and Biochemical Foundations of Living Systems',': Structure and function of proteins and their constituent amino acids');
INSERT INTO Users_topic VALUES(3,'MCAT','Biological and Biochemical Foundations of Living Systems',': Transmission of genetic information from the gene to the protein');
INSERT INTO Users_topic VALUES(4,'MCAT','Biological and Biochemical Foundations of Living Systems',': Transmission of heritable information from generation to generation and the processes that increase genetic diversity');
INSERT INTO Users_topic VALUES(5,'MCAT','Biological and Biochemical Foundations of Living Systems',': Principles of bioenergetics and fuel molecule metabolism');
INSERT INTO Users_topic VALUES(6,'MCAT','Biological and Biochemical Foundations of Living Systems',': Assemblies of molecules, cells, and groups of cells within single cellular and multicellular organisms');
INSERT INTO Users_topic VALUES(7,'MCAT','Biological and Biochemical Foundations of Living Systems',': The structure, growth, physiology, and genetics of prokaryotes and viruses');
INSERT INTO Users_topic VALUES(8,'MCAT','Biological and Biochemical Foundations of Living Systems',': Processes of cell division, differentiation, and specialization');
INSERT INTO Users_topic VALUES(9,'MCAT','Biological and Biochemical Foundations of Living Systems',': Structure and functions of the nervous and endocrine systems and ways these systems coordinate the organ systems');
INSERT INTO Users_topic VALUES(10,'MCAT','Biological and Biochemical Foundations of Living Systems',': Structure and integrative functions of the main organ systems [this includes the respiratory system, circulatory system, lymphatic system, immune system, digestive system, excretory system, reproductive system, muscles, skeletal system, skin system');
INSERT INTO Users_topic VALUES(11,'MCAT','Chemical and Physical Foundations of Biological Systems',': Translational motion, forces, work, energy, and equilibrium in living systems');
INSERT INTO Users_topic VALUES(12,'MCAT','Chemical and Physical Foundations of Biological Systems',': Importance of fluids for the circulation of blood, gas movement, and gas exchange');
INSERT INTO Users_topic VALUES(13,'MCAT','Chemical and Physical Foundations of Biological Systems',': Electrochemistry and electrical circuits and their elements');
INSERT INTO Users_topic VALUES(14,'MCAT','Chemical and Physical Foundations of Biological Systems',': How light and sound interact with matter');
INSERT INTO Users_topic VALUES(15,'MCAT','Chemical and Physical Foundations of Biological Systems',': Atoms, nuclear decay, electronic structure, and atomic chemical behavior');
INSERT INTO Users_topic VALUES(16,'MCAT','Chemical and Physical Foundations of Biological Systems',': Unique nature of water and its solutions');
INSERT INTO Users_topic VALUES(17,'MCAT','Chemical and Physical Foundations of Biological Systems',': Nature of molecules and intermolecular interactions');
INSERT INTO Users_topic VALUES(18,'MCAT','Chemical and Physical Foundations of Biological Systems',': Separation and purification methods');
INSERT INTO Users_topic VALUES(19,'MCAT','Chemical and Physical Foundations of Biological Systems',': Structure, function, and reactivity of biologically relevant molecules');
INSERT INTO Users_topic VALUES(20,'MCAT','Chemical and Physical Foundations of Biological Systems',': Principles of chemical thermodynamics and kinetics');
INSERT INTO Users_topic VALUES(21,'MCAT','Biological, psychological, and sociocultural factors influence the ways that individuals perceive, think about, and react to the world',': Sensing the environment');
INSERT INTO Users_topic VALUES(22,'MCAT','Biological, psychological, and sociocultural factors influence the ways that individuals perceive, think about, and react to the world',': Making sense of the environment');
INSERT INTO Users_topic VALUES(23,'MCAT','Biological, psychological, and sociocultural factors influence the ways that individuals perceive, think about, and react to the world',': Responding to the world');
INSERT INTO Users_topic VALUES(24,'MCAT','Biological, psychological, and sociocultural factors influence behavior and behavior change',': Individual influences on behavior');
INSERT INTO Users_topic VALUES(25,'MCAT','Biological, psychological, and sociocultural factors influence behavior and behavior change',': Social processes that influence human behavior');
INSERT INTO Users_topic VALUES(26,'MCAT','Biological, psychological, and sociocultural factors influence behavior and behavior change',': Attitude and behavior change');
INSERT INTO Users_topic VALUES(27,'MCAT','Psychological, sociocultural, and biological factors influence the way we think about ourselves and others, as well as how we interact with others',': Self-identity');
INSERT INTO Users_topic VALUES(28,'MCAT','Psychological, sociocultural, and biological factors influence the way we think about ourselves and others, as well as how we interact with others',': Social thinking');
INSERT INTO Users_topic VALUES(29,'MCAT','Psychological, sociocultural, and biological factors influence the way we think about ourselves and others, as well as how we interact with others',': Social interactions');
INSERT INTO Users_topic VALUES(30,'MCAT','Cultural and social differences influence well-being',': Understanding social structure');
INSERT INTO Users_topic VALUES(31,'MCAT','Cultural and social differences influence well-being',': Demographic characteristics and processes');
INSERT INTO Users_topic VALUES(32,'MCAT','Social stratification and access to resources influence well-being',': Social inequality');
CREATE TABLE IF NOT EXISTS "Users_question" ("QUESTION_ID" integer NOT NULL PRIMARY KEY, "OPTION_ONE" varchar(1000) NOT NULL, "OPTION_TWO" varchar(1000) NOT NULL, "OPTION_THREE" varchar(1000) NOT NULL, "OPTION_FOUR" varchar(1000) NOT NULL, "EXPLANATION_ONE" text NOT NULL, "EXPLANATION_TWO" text NOT NULL, "EXPLANATION_THREE" text NOT NULL, "EXPLANATION_FOUR" text NOT NULL, "CORRECT_ANSWER" smallint NOT NULL, "TOPIC_id" integer NOT NULL REFERENCES "Users_topic" ("TOPIC_ID") DEFERRABLE INITIALLY DEFERRED, "QUESTION_NAME" text NOT NULL);
INSERT INTO Users_question VALUES(3,'{''1. Valine''}','{''2. Leucine''}','{''3. Glutamic acid''}','{''4. Phenylalanine''}','{''Incorrect. Valine is an aliphatic, nonpolar amino acid. Nonpolar amino acids tend to cluster in the interior of proteins to avoid contact with water. A common mistake is to forget that valine is nonpolar and associate it with some sort of polarity because it has no charge; therefore, a person may think that it will be found on the surface interacting with the water.''}','{''Incorrect. Leucine is another aliphatic, nonpolar amino acid. Like valine, it prefers to reside in the hydrophobic core of the protein. Students can be confused as to where certain amino acids are found within proteins, but it must be remembered that nonpolar amino acids tend to be buried within the protein structure away from the aqueous environment. ''}','{''Correct. Glutamic acid is a negatively charged, polar amino acid at physiological pH. Charged and polar amino acids are hydrophilic, readily interacting with water and thus are commonly found on the protein surface. A common pitfall is not remembering the charge or polarity of the amino acids and thinking that it may be buried within the protein structure.''}','{"Incorrect. Phenylalanine contains a large, hydrophobic aromatic ring. Due to its nonpolar nature, it''s usually found in the interior of the protein, avoiding the aqueous environment. Although some aromatic residues can be found on the surface, phenylalanine''s overall hydrophobicity makes it less likely than a charged amino acid like glutamic acid. Students may consider aromatic rings to have some polar character because of the pi electrons; however, in phenylalanine, the ring is predominantly nonpolar."}',3,2,'Which of the following amino acids is MOST likely to be found on the surface of a protein in an aqueous solution?');
INSERT INTO Users_question VALUES(4,'{''1.  Compare the self-reported happiness levels of residents who were displaced with those who remained in the neighborhood.''}','{''2.  Analyze publicly available health records (e.g., hospital visits, prescriptions) of displaced residents before and after the gentrification occurred, and compare the changes to a control group of similar individuals who did not experience displacement.''}','{''3.  Conduct interviews with community leaders to gather their perspectives on the impact of gentrification on the well-being of residents.''}','{''4.  Track the employment rates of displaced residents one year after the gentrification occurred and compare them to the national average employment rate.''}','{"This option is weak because self-reported happiness is a subjective measure and may be influenced by factors other than displacement, such as personality traits or current life events.  It doesn''t account for pre-displacement happiness levels. It also doesn''t control for potential confounding variables. A common pitfall is assuming that a subjective measure is sufficient to demonstrate a causal relationship. Furthermore, even if the displaced residents report lower happiness, it doesn''t necessarily mean it''s *caused* by the displacement; it could be correlated for other reasons.  A stronger design would involve more objective and longitudinal measures."}','{''This is the STRONGEST option. Analyzing health records provides a more objective measure of well-being than self-reported happiness.  Comparing pre- and post-displacement health records for the displaced group allows researchers to assess changes within the same individuals. Crucially, including a control group of similar individuals who did not experience displacement helps to isolate the effect of displacement from other factors that might influence health outcomes over time. This design controls for confounding variables and provides a basis for causal inference.  This design minimizes bias and provides a robust comparison.''}','{''This option relies on qualitative data from community leaders, which can be valuable for understanding the broader context but is not sufficient to establish a causal relationship between displacement and well-being. Community leader perspectives might be biased or incomplete.  It lacks objective measures and a control group, making it difficult to determine whether any observed changes are directly attributable to displacement.  A pitfall is relying solely on subjective perspectives without empirical data. This research is helpful in creating a hypothesis but not testing one.''}','{''This option only focuses on one aspect of well-being (employment) and does not provide a comprehensive assessment of overall well-being. Comparing to the national average is problematic because the displaced residents might have had different pre-existing employment rates or face different local labor market conditions than the national average.  It also lacks a control group and does not consider pre-displacement employment rates of the displaced group. A common pitfall is using an inappropriate comparison group (national average) rather than a more relevant control group (similar individuals who were not displaced) and a single factor does not tell the whole story.''}',2,32,'Researchers are studying the effects of gentrification on a low-income neighborhood. They find that after a new luxury apartment complex is built, rents in the surrounding area increase significantly, leading to displacement of many long-term residents. The researchers hypothesize that this displacement leads to a decrease in overall well-being for the displaced residents. To test this hypothesis, which of the following research designs would provide the STRONGEST evidence?');
INSERT INTO Users_question VALUES(5,'{''1) Valine to Alanine''}','{''2) Lysine to Arginine''}','{''3) Serine to Phenylalanine''}','{''4) Glutamate to Aspartate''}','{"Incorrect. Both valine and alanine are nonpolar, hydrophobic amino acids. While alanine is smaller than valine, the overall hydrophobic nature of the side chain is maintained. This conservative substitution is less likely to drastically disrupt protein folding or cause aggregation, as the hydrophobic interactions driving protein folding would still be largely intact.  A common pitfall is assuming any change will drastically alter the protein''s behavior; sometimes, the changes are subtle, and the protein can still function normally, or near-normally."}','{"Incorrect. Both lysine and arginine are positively charged, basic amino acids.  While arginine is slightly bulkier than lysine, they both can form similar ionic and hydrogen bond interactions within the protein. This is another conservative substitution. Replacing one positively charged amino acid with another is unlikely to have a major effect on protein folding. Students might incorrectly assume that ''changing the charge'' is always a big deal, but here, the charge is maintained, so the protein''s electrostatic interactions are not significantly disrupted."}','{"Correct. Serine is a polar, hydrophilic amino acid, while phenylalanine is a large, nonpolar, hydrophobic amino acid. Replacing a polar, hydrophilic amino acid with a large, nonpolar, hydrophobic one introduces a significant change in the amino acid''s properties. This substitution is most likely to disrupt the protein''s folding and lead to aggregation, because now the protein will likely be unable to form hydrogen bonds with the aqueous environment, causing the protein to collapse onto itself. The bulky hydrophobic side chain of phenylalanine will disrupt the existing hydrogen bonding network and promote hydrophobic interactions that could lead to misfolding and aggregation. This is a non-conservative substitution. Pay close attention to the hydrophobicity/hydrophilicity changes."}','{"Incorrect. Both glutamate and aspartate are negatively charged, acidic amino acids. Aspartate is simply a shorter version of glutamate. This is a conservative substitution; while there may be some subtle shifts in the protein''s structure, the negative charge is retained, and the side chains are still relatively small.  The protein''s ability to form ionic bonds and hydrogen bonds at that location would likely be maintained. Students may overthink the importance of the slight size difference between these two amino acids, missing the key similarity in their charge and polarity."}',3,2,'A researcher is studying a novel protein that appears to be misfolding and aggregating in the cell, leading to cellular dysfunction. They hypothesize that a specific amino acid substitution in the protein''s primary sequence is responsible for this misfolding. To investigate this, they create several mutant versions of the protein, each with a single amino acid change at the suspected site. They then express these mutant proteins in cells and assess their folding and aggregation properties. Which of the following amino acid substitutions would be MOST likely to disrupt the protein''s folding and lead to aggregation?');
INSERT INTO Users_question VALUES(6,'{''1. Glutamic acid''}','{''2. Lysine''}','{''3. Valine''}','{''4. Histidine''}','{''Incorrect. Glutamic acid is a negatively charged (acidic) amino acid. Charged amino acids are hydrophilic and prefer to interact with aqueous environments. They are unlikely to be found in the hydrophobic core of a transmembrane domain. A common pitfall is not recognizing the properties of glutamic acid (charged, hydrophilic) and how those properties dictate its location within a protein.''}','{"Incorrect. Lysine is a positively charged (basic) amino acid. Like glutamic acid, it''s hydrophilic and prefers aqueous environments. It would be energetically unfavorable for lysine to reside within the hydrophobic transmembrane region. A common pitfall is forgetting the basic amino acids and their hydrophilic nature."}','{''Correct. Valine is a nonpolar, hydrophobic amino acid. Transmembrane domains span the lipid bilayer of a cell membrane, which is a highly hydrophobic environment. Hydrophobic amino acids like valine are favored in this region because they minimize unfavorable interactions with the hydrophobic lipid tails. This interaction stabilizes the protein within the membrane. A common pitfall is overlooking the importance of hydrophobicity in transmembrane domains.''}','{"Incorrect. Histidine can be charged or uncharged depending on the pH. While it can sometimes be found near the membrane surface where it can participate in interactions with the polar head groups of lipids, it is less likely to be buried deeply within the transmembrane domain compared to a strictly hydrophobic amino acid like valine. Histidine''s imidazole side chain has a pKa close to physiological pH, making it more likely to be involved in catalysis or proton transfer at the protein surface. A common pitfall is thinking that because histidine *can* be uncharged under certain conditions, it''s a good candidate for a transmembrane domain, without considering how pH affects its protonation state, and the presence of more favorable hydrophobic options."}',3,2,'Which of the following amino acids is most likely to be found in the transmembrane domain of a protein?');
INSERT INTO Users_question VALUES(7,'{''1. E. coli cells that have not been transformed with any plasmid.''}','{''2. E. coli cells transformed with a plasmid containing the Foldase gene with a premature stop codon early in the sequence.''}','{''3. E. coli cells transformed with a plasmid containing the wild-type (non-mutated) Foldase gene.''}','{''4. Purified, commercially available bovine serum albumin (BSA).''}','{"Incorrect. While using untransformed E. coli cells can provide a baseline for cellular components, it doesn''t account for the effects of expressing the Foldase protein itself (unmutated). The CD signal from untransformed cells will be very different from that of cells expressing Foldase, making it difficult to isolate the impact of the mutations. Many cellular proteins have secondary structure and would obscure the signal from Foldase. This option fails to isolate the variable of interest (the mutations) and does not control for Foldase expression."}','{''Incorrect. This control would result in truncated protein, which would most likely be degraded by the cell. Even if not degraded the protein would be too short to form any secondary structure or have a dramatically different secondary structure content than the full length Foldase protein. The goal is to maintain the protein under the same conditions without the mutation.''}','{"Correct. The most appropriate control group is E. coli cells transformed with a plasmid containing the wild-type Foldase gene. This control ensures that the baseline folding and CD spectrum of the normal, functional protein are established. By comparing the CD spectra of the mutated Foldase proteins to that of the wild-type Foldase, the researcher can accurately determine how each mutation affects the protein''s secondary structure and, by inference, its overall folding. This control accounts for the effects of expressing the Foldase protein, the presence of the plasmid, and any other factors related to the expression and purification process, leaving the mutation as the sole variable. This is the standard to which the mutated proteins are compared."}','{''Incorrect. BSA is a different protein altogether and will have a distinct amino acid sequence and secondary structure composition compared to Foldase. While BSA is often used as a protein standard, it does not control for any of the factors specific to Foldase expression, purification, or inherent folding properties. Comparing the CD spectrum of mutated Foldase to that of BSA would not provide meaningful information about the impact of the mutations on Foldase folding. The differences observed would be due to the inherent differences between the two proteins, not the mutations.''}',3,2,replace('A researcher is studying the effect of mutations on the folding of a protein called ''Foldase''. Foldase normally folds into a specific tertiary structure crucial for its function. The researcher introduces several point mutations in the Foldase gene, expresses the mutated proteins in E. coli, and then purifies them. To assess the impact of each mutation on protein folding, the researcher uses circular dichroism (CD) spectroscopy. CD spectroscopy measures the difference in absorbance of left- and right-circularly polarized light, which is sensitive to the secondary structure content of proteins. \n\nWhich of the following would be the MOST appropriate control group for this experiment?','\n',char(10)));
INSERT INTO Users_question VALUES(8,'{''1. Alanine to Valine''}','{''2. Lysine to Arginine''}','{''3. Glycine to Proline''}','{''4. Serine to Threonine''}','{''Alanine and Valine are both nonpolar, aliphatic amino acids. While Valine is slightly more bulky than Alanine due to its branched side chain, this substitution is LESS likely to disrupt the alpha-helix significantly compared to introducing proline or charged residues. The change in steric hindrance is not drastic enough to destabilize the helix substantially. Many students might pick this because they recognize that valine is bulkier, but they underestimate the impact of proline or charged residues on secondary structure.''}','{''Lysine and Arginine are both positively charged, basic amino acids. While their side chains differ in structure (lysine has an alkyl chain with an amine, arginine has a guanidinium group), they share similar properties in terms of charge and polarity. This substitution is LESS likely to disrupt the alpha-helix because the overall electrostatic environment is maintained. Students often think of charge as universally disruptive, but maintaining charge type is less destabilizing than changing the charge or introducing proline. This is a distractor.''}','{''Proline is a unique amino acid because its side chain cyclizes and bonds to the nitrogen atom of the amino group, creating a rigid ring structure. This rigidity restricts the conformational flexibility of the peptide backbone and disrupts the regular hydrogen bonding pattern required for alpha-helix formation. Glycine, on the other hand, is very flexible due to its small side chain (just a hydrogen atom) and is often found in turns and loops. Substituting Glycine with Proline introduces significant steric hindrance and disrupts the helix. This is the MOST disruptive substitution among the choices. Many students might overlook the impact of proline on secondary structure, particularly its effect on the alpha-helix. The unique cyclic structure of proline is the key.''}','{"Serine and Threonine are both polar, uncharged amino acids containing hydroxyl (-OH) groups. They are similar in size and polarity. This substitution is LESS likely to significantly disrupt the alpha-helix structure, as the hydrogen bonding potential is largely maintained. The slightly larger methyl group in threonine compared to serine might cause a minor steric effect, but it is not as disruptive as proline. Students might get confused and think Threonine''s slightly bulkier side chain causes disruption, but the effect is minimal compared to the effect of Proline"}',3,2,'A researcher is studying a protein involved in muscle contraction. They discover that a mutation causes a single amino acid substitution within a region of the protein known to be an alpha-helix. Which of the following amino acid substitutions is MOST likely to disrupt the alpha-helical structure?');
INSERT INTO Users_question VALUES(9,'{''1.  AA* significantly improves protein folding at low concentrations (10 µM) compared to the control (0 µM).''}','{''2.  AA* has no effect on protein folding, as the percentage of correctly folded proteins remains relatively constant across all concentrations.''}','{''3.  AA* likely inhibits protein folding in a concentration-dependent manner.''}','{''4.  The optimal concentration of AA* for protein folding is 20 µM, as it has the lowest standard deviation.''}','{"Incorrect. While the percentage of correctly folded proteins is slightly lower at 10 µM AA* compared to 0 µM, the standard deviations overlap significantly (3.2 vs 3.0). This means the difference observed could easily be due to random chance and isn''t statistically significant. A common pitfall is to interpret small numerical differences as meaningful without considering the variability within the data (as indicated by the standard deviation). One would need to perform a statistical test (like a t-test) to determine if the difference is actually significant. Without that, you can''t definitively say there''s a real improvement."}','{''Incorrect. A clear trend of decreasing correctly folded proteins is visible as the AA* concentration increases.  The percentage drops from 85% at 0 µM to 65% at 40 µM. Even though the standard deviations are present, the overall trend demonstrates an effect. While the change between adjacent concentrations might not always be significant on its own, the cumulative effect suggests a real impact. The pitfall here is focusing too much on the standard deviations of individual points and missing the overall trend in the data. Visualizing the data as a graph could help avoid this error.''}','{''Correct. The data clearly shows a decreasing trend in the percentage of correctly folded proteins as the concentration of AA* increases. This suggests that AA* interferes with the protein folding process, and this interference becomes more pronounced at higher concentrations. The trend is consistent despite the standard deviations. A decreasing trend with increasing concentration strongly implies an inhibitory effect that is concentration-dependent. This option is the most justified interpretation of the data provided, as the trend is clear and consistent.''}','{"Incorrect. While 20 µM does have the lowest standard deviation, the percentage of correctly folded proteins at this concentration (78%) is lower than at lower AA* concentrations (0 µM and 10 µM). The goal is typically to maximize the percentage of correctly folded proteins. Choosing a concentration based solely on the lowest standard deviation, while ignoring the actual outcome (percentage folded), is a misinterpretation of the data. Standard deviation reflects precision, but not necessarily accuracy or effectiveness in achieving the desired result. Lower standard deviation means more consistent result, it doesn''t mean best result"}',3,2,replace('A researcher is studying the effects of a novel amino acid analog (AA*) on protein folding. They introduce AA* into a bacterial culture and measure the percentage of correctly folded proteins at different concentrations of AA*. The following data is obtained:\n\n| AA* Concentration (µM) | % Correctly Folded Proteins | Standard Deviation |\n|--------------------------|-----------------------------|----------------------|\n| 0                        | 85                          | 3.2                  |\n| 10                       | 82                          | 3.0                  |\n| 20                       | 78                          | 2.8                  |\n| 30                       | 73                          | 3.5                  |\n| 40                       | 65                          | 3.1                  |\n\nBased on the data, which of the following statements is MOST justified regarding the effect of AA* on protein folding?\n','\n',char(10)));
INSERT INTO Users_question VALUES(10,'{''1.  AA* significantly improves protein folding at low concentrations (10 µM) compared to the control (0 µM).''}','{''2.  AA* has no effect on protein folding, as the percentage of correctly folded proteins remains relatively constant across all concentrations.''}','{''3.  AA* likely inhibits protein folding in a concentration-dependent manner.''}','{''4.  The optimal concentration of AA* for protein folding is 20 µM, as it has the lowest standard deviation.''}','{"Incorrect. While the percentage of correctly folded proteins is slightly lower at 10 µM AA* compared to 0 µM, the standard deviations overlap significantly (3.2 vs 3.0). This means the difference observed could easily be due to random chance and isn''t statistically significant. A common pitfall is to interpret small numerical differences as meaningful without considering the variability within the data (as indicated by the standard deviation). One would need to perform a statistical test (like a t-test) to determine if the difference is actually significant. Without that, you can''t definitively say there''s a real improvement."}','{''Incorrect. A clear trend of decreasing correctly folded proteins is visible as the AA* concentration increases.  The percentage drops from 85% at 0 µM to 65% at 40 µM. Even though the standard deviations are present, the overall trend demonstrates an effect. While the change between adjacent concentrations might not always be significant on its own, the cumulative effect suggests a real impact. The pitfall here is focusing too much on the standard deviations of individual points and missing the overall trend in the data. Visualizing the data as a graph could help avoid this error.''}','{''Correct. The data clearly shows a decreasing trend in the percentage of correctly folded proteins as the concentration of AA* increases. This suggests that AA* interferes with the protein folding process, and this interference becomes more pronounced at higher concentrations. The trend is consistent despite the standard deviations. A decreasing trend with increasing concentration strongly implies an inhibitory effect that is concentration-dependent. This option is the most justified interpretation of the data provided, as the trend is clear and consistent.''}','{"Incorrect. While 20 µM does have the lowest standard deviation, the percentage of correctly folded proteins at this concentration (78%) is lower than at lower AA* concentrations (0 µM and 10 µM). The goal is typically to maximize the percentage of correctly folded proteins. Choosing a concentration based solely on the lowest standard deviation, while ignoring the actual outcome (percentage folded), is a misinterpretation of the data. Standard deviation reflects precision, but not necessarily accuracy or effectiveness in achieving the desired result. Lower standard deviation means more consistent result, it doesn''t mean best result"}',3,2,replace('A researcher is studying the effects of a novel amino acid analog (AA*) on protein folding. They introduce AA* into a bacterial culture and measure the percentage of correctly folded proteins at different concentrations of AA*. The following data is obtained:\n\n| AA* Concentration (µM) | % Correctly Folded Proteins | Standard Deviation |\n|--------------------------|-----------------------------|----------------------|\n| 0                        | 85                          | 3.2                  |\n| 10                       | 82                          | 3.0                  |\n| 20                       | 78                          | 2.8                  |\n| 30                       | 73                          | 3.5                  |\n| 40                       | 65                          | 3.1                  |\n\nBased on the data, which of the following statements is MOST justified regarding the effect of AA* on protein folding?\n','\n',char(10)));
INSERT INTO Users_question VALUES(11,'{"The inhibitor decreases the enzyme''s catalytic efficiency by 50%."}','{"The inhibitor increases the enzyme''s catalytic efficiency by 100%."}','{"The inhibitor has no effect on the enzyme''s catalytic efficiency because Vmax remains constant."}','{"The inhibitor decreases the enzyme''s catalytic efficiency by 10%."}','{"This is the correct answer. Catalytic efficiency is represented by the ratio Vmax/Km. Without the inhibitor, catalytic efficiency is 100/10 = 10. With the inhibitor, it''s 100/20 = 5.  The efficiency is halved (decreased by 50%). This option correctly calculates and interprets the change in catalytic efficiency based on the provided data."}','{''This is incorrect. The Km value increased in the presence of the inhibitor, indicating a lower affinity of the enzyme for the substrate, which would reduce catalytic efficiency, not increase it. Some test takers may confuse the increase in Km with an increase in efficiency. Competitive inhibitors increase Km, which is inversely related to catalytic efficiency.''}','{''This is incorrect. While Vmax does remain constant in the presence of a competitive inhibitor, catalytic efficiency is defined by the ratio of Vmax to Km. Since Km changes, the catalytic efficiency does indeed change. This option ignores the effect of Km on catalytic efficiency, only focusing on Vmax.''}','{''This is incorrect. While the Km increased by a factor of two, the catalytic efficiency is calculated based on the ratio of Vmax to Km. A 10% decrease does not align with the given data. A common mistake is to focus solely on the change in Km, but without using it to correctly compute the ratio of Vmax and Km.''}',1,2,replace('A researcher is studying the effect of a novel competitive inhibitor on the enzyme kinetics of an important metabolic enzyme. They perform Michaelis-Menten kinetics experiments both in the presence and absence of the inhibitor. The following data is obtained:\n\n| Condition             | Vmax (µmol/min) | Km (µM) |\n|-----------------------|-----------------|---------|\n| No Inhibitor          | 100             | 10      |\n| With Inhibitor        | 100             | 20      |\n\nBased on this data, what is the best conclusion about the inhibitor''s effect on the enzyme''s catalytic efficiency?\n','\n',char(10)));
INSERT INTO Users_question VALUES(12,'{"The inhibitor decreases the enzyme''s catalytic efficiency by 50%."}','{"The inhibitor increases the enzyme''s catalytic efficiency by 100%."}','{"The inhibitor has no effect on the enzyme''s catalytic efficiency because Vmax remains constant."}','{"The inhibitor decreases the enzyme''s catalytic efficiency by 10%."}','{"This is the correct answer. Catalytic efficiency is represented by the ratio Vmax/Km. Without the inhibitor, catalytic efficiency is 100/10 = 10. With the inhibitor, it''s 100/20 = 5.  The efficiency is halved (decreased by 50%). This option correctly calculates and interprets the change in catalytic efficiency based on the provided data."}','{''This is incorrect. The Km value increased in the presence of the inhibitor, indicating a lower affinity of the enzyme for the substrate, which would reduce catalytic efficiency, not increase it. Some test takers may confuse the increase in Km with an increase in efficiency. Competitive inhibitors increase Km, which is inversely related to catalytic efficiency.''}','{''This is incorrect. While Vmax does remain constant in the presence of a competitive inhibitor, catalytic efficiency is defined by the ratio of Vmax to Km. Since Km changes, the catalytic efficiency does indeed change. This option ignores the effect of Km on catalytic efficiency, only focusing on Vmax.''}','{''This is incorrect. While the Km increased by a factor of two, the catalytic efficiency is calculated based on the ratio of Vmax to Km. A 10% decrease does not align with the given data. A common mistake is to focus solely on the change in Km, but without using it to correctly compute the ratio of Vmax and Km.''}',1,2,replace('A researcher is studying the effect of a novel competitive inhibitor on the enzyme kinetics of an important metabolic enzyme. They perform Michaelis-Menten kinetics experiments both in the presence and absence of the inhibitor. The following data is obtained:\n\n| Condition             | Vmax (µmol/min) | Km (µM) |\n|-----------------------|-----------------|---------|\n| No Inhibitor          | 100             | 10      |\n| With Inhibitor        | 100             | 20      |\n\nBased on this data, what is the best conclusion about the inhibitor''s effect on the enzyme''s catalytic efficiency?\n','\n',char(10)));
INSERT INTO Users_question VALUES(13,'{''1. A high proportion of proline residues that introduce kinks in the polypeptide chain, disrupting regular secondary structures.''}','{''2. Numerous cysteine residues that form disulfide bonds, cross-linking different parts of the protein and stabilizing its structure.''}','{''3. A large number of amino acids with charged side chains (e.g., glutamate, lysine) that promote electrostatic repulsion and prevent aggregation.''}','{''4. A predominance of small, hydrophobic amino acids (e.g., alanine, valine) that minimize interactions with the surrounding aqueous environment.''}','{"Incorrect. Proline is known to disrupt alpha-helices and beta-sheets due to its rigid cyclic structure and its inability to form hydrogen bonds with the backbone amide hydrogen. If there were a high proportion of proline, the protein would be *less* stable and *more* prone to denaturation, not more resistant. A common pitfall is misremembering proline''s role; it''s a helix breaker, not a stabilizer, so don''t assume rigidity always leads to stability in this context."}','{"Correct. Disulfide bonds, formed between the sulfur atoms of cysteine residues, are covalent bonds that can cross-link different parts of a polypeptide chain or link separate polypeptide chains. These bonds significantly increase the stability of a protein by holding its structure together more tightly, making it more resistant to denaturation. While the question mentions limited tertiary structure, the existence of even a few strategically placed disulfide bridges could dramatically increase stability. Many students might overlook this if they''re overly focused on the lack of tertiary structure, failing to recognize the powerful stabilizing effect of covalent cross-links."}','{''Incorrect. While charged amino acids contribute to protein solubility and can prevent aggregation in an aqueous environment, electrostatic repulsion does not inherently increase the *stability* of the protein structure against denaturation. In fact, excessive charge repulsion could potentially destabilize the folded state if not properly balanced by other stabilizing interactions. The prompt refers to resistance to denaturation, so electrostatic repulsion will do very little. Students selecting this answer might be confusing protein solubility with protein stability; remember that preventing aggregation is different from preventing unfolding.''}','{"Incorrect. While hydrophobic interactions are important for protein folding and stability, a predominance of small, hydrophobic amino acids alone wouldn''t necessarily explain the *exceptional* resistance to denaturation described in the question. Hydrophobic interactions contribute to the overall stability, but are relatively weak compared to covalent disulfide bonds. This option may appeal to those who overemphasize the hydrophobic effect, failing to consider the significance of covalent cross-linking in enhancing stability, especially against harsh conditions. The small size of the amino acids also doesn''t inherently explain the resilience as there would be very little stabilizing Van Der Waals forces as a result. Also it fails to explain the alpha helix/beta sheet composition."}',2,2,'A researcher is studying a novel protein and discovers that it is primarily composed of alpha-helices and beta-sheets, with very little tertiary structure. The protein is found to be highly resistant to denaturation by heat or chemical agents. Which of the following amino acid characteristics would MOST likely contribute to this protein''s unusual stability?');
INSERT INTO Users_question VALUES(14,'{''1. A high proportion of proline residues that introduce kinks in the polypeptide chain, disrupting regular secondary structures.''}','{''2. Numerous cysteine residues that form disulfide bonds, cross-linking different parts of the protein and stabilizing its structure.''}','{''3. A large number of amino acids with charged side chains (e.g., glutamate, lysine) that promote electrostatic repulsion and prevent aggregation.''}','{''4. A predominance of small, hydrophobic amino acids (e.g., alanine, valine) that minimize interactions with the surrounding aqueous environment.''}','{"Incorrect. Proline is known to disrupt alpha-helices and beta-sheets due to its rigid cyclic structure and its inability to form hydrogen bonds with the backbone amide hydrogen. If there were a high proportion of proline, the protein would be *less* stable and *more* prone to denaturation, not more resistant. A common pitfall is misremembering proline''s role; it''s a helix breaker, not a stabilizer, so don''t assume rigidity always leads to stability in this context."}','{"Correct. Disulfide bonds, formed between the sulfur atoms of cysteine residues, are covalent bonds that can cross-link different parts of a polypeptide chain or link separate polypeptide chains. These bonds significantly increase the stability of a protein by holding its structure together more tightly, making it more resistant to denaturation. While the question mentions limited tertiary structure, the existence of even a few strategically placed disulfide bridges could dramatically increase stability. Many students might overlook this if they''re overly focused on the lack of tertiary structure, failing to recognize the powerful stabilizing effect of covalent cross-links."}','{''Incorrect. While charged amino acids contribute to protein solubility and can prevent aggregation in an aqueous environment, electrostatic repulsion does not inherently increase the *stability* of the protein structure against denaturation. In fact, excessive charge repulsion could potentially destabilize the folded state if not properly balanced by other stabilizing interactions. The prompt refers to resistance to denaturation, so electrostatic repulsion will do very little. Students selecting this answer might be confusing protein solubility with protein stability; remember that preventing aggregation is different from preventing unfolding.''}','{"Incorrect. While hydrophobic interactions are important for protein folding and stability, a predominance of small, hydrophobic amino acids alone wouldn''t necessarily explain the *exceptional* resistance to denaturation described in the question. Hydrophobic interactions contribute to the overall stability, but are relatively weak compared to covalent disulfide bonds. This option may appeal to those who overemphasize the hydrophobic effect, failing to consider the significance of covalent cross-linking in enhancing stability, especially against harsh conditions. The small size of the amino acids also doesn''t inherently explain the resilience as there would be very little stabilizing Van Der Waals forces as a result. Also it fails to explain the alpha helix/beta sheet composition."}',2,2,'A researcher is studying a novel protein and discovers that it is primarily composed of alpha-helices and beta-sheets, with very little tertiary structure. The protein is found to be highly resistant to denaturation by heat or chemical agents. Which of the following amino acid characteristics would MOST likely contribute to this protein''s unusual stability?');
INSERT INTO Users_question VALUES(15,'{''1. Valine''}','{''2. Leucine''}','{''3. Glutamic acid''}','{''4. Phenylalanine''}','{''Valine is a nonpolar, hydrophobic amino acid.  Its side chain consists of an isopropyl group. Hydrophobic amino acids tend to cluster in the interior of proteins to avoid contact with water. Therefore, valine is unlikely to be found on the surface of a protein in an aqueous environment. Many students incorrectly assume that all amino acids are equally likely to be found on the surface.''}','{''Leucine is another nonpolar, hydrophobic amino acid with an isobutyl side chain. Similar to valine, leucine prefers the hydrophobic core of a protein to minimize contact with water. Thus, leucine is not typically found on the surface of a protein in an aqueous environment. Students often forget to distinguish between hydrophobic and hydrophilic amino acids, leading them to consider hydrophobic options incorrectly.''}','{"Glutamic acid is a polar, negatively charged (acidic) amino acid at physiological pH. Due to its charged side chain (a carboxylate group), it is highly soluble in water and interacts favorably with aqueous environments. Therefore, glutamic acid is very likely to be found on the surface of a protein exposed to water. Some students might hesitate because of the ''acid'' in the name, but its behavior at physiological pH makes it a surface-preferring residue. This is the correct answer."}','{''Phenylalanine is a nonpolar, hydrophobic amino acid containing a benzene ring. Its bulky, hydrophobic side chain strongly prefers the interior of proteins, far from the aqueous environment. Phenylalanine on the surface would disrupt the favorable interactions between the protein and water. Students might pick this if they focus only on the aromatic ring and forget about its overall hydrophobicity.''}',3,2,'Which of the following amino acids is MOST likely to be found on the surface of a protein in an aqueous environment?');
INSERT INTO Users_question VALUES(16,'{''1. Valine''}','{''2. Leucine''}','{''3. Glutamic acid''}','{''4. Phenylalanine''}','{''Valine is a nonpolar, hydrophobic amino acid.  Its side chain consists of an isopropyl group. Hydrophobic amino acids tend to cluster in the interior of proteins to avoid contact with water. Therefore, valine is unlikely to be found on the surface of a protein in an aqueous environment. Many students incorrectly assume that all amino acids are equally likely to be found on the surface.''}','{''Leucine is another nonpolar, hydrophobic amino acid with an isobutyl side chain. Similar to valine, leucine prefers the hydrophobic core of a protein to minimize contact with water. Thus, leucine is not typically found on the surface of a protein in an aqueous environment. Students often forget to distinguish between hydrophobic and hydrophilic amino acids, leading them to consider hydrophobic options incorrectly.''}','{"Glutamic acid is a polar, negatively charged (acidic) amino acid at physiological pH. Due to its charged side chain (a carboxylate group), it is highly soluble in water and interacts favorably with aqueous environments. Therefore, glutamic acid is very likely to be found on the surface of a protein exposed to water. Some students might hesitate because of the ''acid'' in the name, but its behavior at physiological pH makes it a surface-preferring residue. This is the correct answer."}','{''Phenylalanine is a nonpolar, hydrophobic amino acid containing a benzene ring. Its bulky, hydrophobic side chain strongly prefers the interior of proteins, far from the aqueous environment. Phenylalanine on the surface would disrupt the favorable interactions between the protein and water. Students might pick this if they focus only on the aromatic ring and forget about its overall hydrophobicity.''}',3,2,'Which of the following amino acids is MOST likely to be found on the surface of a protein in an aqueous environment?');
INSERT INTO Users_question VALUES(17,'{''1. Incubating the enzyme with the wild-type protein in the presence of a phosphatase inhibitor.''}','{''2. Incubating the enzyme with a non-phosphorylatable protein substrate.''}','{''3. Incubating the modified proteins with the kinase buffer without the enzyme.''}','{''4. Measuring the enzyme activity at different pH levels to determine the optimal pH.''}','{"Incorrect. While phosphatase inhibitors are useful for preventing dephosphorylation of the target protein, they don''t address the core question of substrate specificity. The experiment aims to determine which amino acid residues are crucial for enzyme-substrate interaction, not to prevent the reverse reaction. This option focuses on a general experimental consideration but misses the key element of controlling for inherent enzymatic activity in the absence of specific substrates."}','{"Incorrect. Using a non-phosphorylatable substrate provides information about the enzyme''s ability to bind, but not phosphorylate, other proteins. It can help assess the enzyme''s general affinity for protein substrates or rule out non-specific interactions. However, it does not isolate the effect of specific amino acid substitutions on the target protein''s phosphorylation rate. The question focuses on understanding how specific amino acids of the target protein play a role in phosphorylation, not general binding."}','{"Correct. This control is crucial to determine if the observed phosphorylation is solely due to the enzyme''s activity and not due to any other factors, such as auto-phosphorylation or contaminants present in the buffer or modified proteins. By excluding the enzyme, the researcher can establish a baseline level of phosphorylation for each modified protein, ensuring that any increase in phosphorylation upon enzyme addition is directly attributable to the enzyme''s specific activity on that substrate. This control accounts for background noise or non-enzyme-mediated phosphorylation, which is essential for accurate interpretation of the results."}','{"Incorrect. Determining the optimal pH is important for characterizing the enzyme''s activity in general, but it doesn''t directly control for the specificity of the enzyme towards different modified protein substrates. While pH can influence enzyme activity, it doesn''t isolate the effect of amino acid substitutions on the phosphorylation rate, which is the primary focus of the experiment. This option addresses optimization of enzyme conditions, rather than directly controlling for the effects of substrate modifications on enzyme activity."}',3,2,'A researcher is studying a novel enzyme that catalyzes the phosphorylation of a specific protein in a signaling pathway. To investigate the enzyme''s substrate specificity, the researcher designs an experiment where they incubate the enzyme with several modified versions of the target protein. Each modified protein has a single amino acid substitution at a different location within the protein sequence. The researcher measures the rate of phosphorylation for each modified protein and compares it to the rate observed with the wild-type protein. Which of the following is the MOST significant control the researcher should include in this experiment to ensure the validity of their conclusions regarding substrate specificity?');
INSERT INTO Users_question VALUES(18,'{''1. Incubating the enzyme with the wild-type protein in the presence of a phosphatase inhibitor.''}','{''2. Incubating the enzyme with a non-phosphorylatable protein substrate.''}','{''3. Incubating the modified proteins with the kinase buffer without the enzyme.''}','{''4. Measuring the enzyme activity at different pH levels to determine the optimal pH.''}','{"Incorrect. While phosphatase inhibitors are useful for preventing dephosphorylation of the target protein, they don''t address the core question of substrate specificity. The experiment aims to determine which amino acid residues are crucial for enzyme-substrate interaction, not to prevent the reverse reaction. This option focuses on a general experimental consideration but misses the key element of controlling for inherent enzymatic activity in the absence of specific substrates."}','{"Incorrect. Using a non-phosphorylatable substrate provides information about the enzyme''s ability to bind, but not phosphorylate, other proteins. It can help assess the enzyme''s general affinity for protein substrates or rule out non-specific interactions. However, it does not isolate the effect of specific amino acid substitutions on the target protein''s phosphorylation rate. The question focuses on understanding how specific amino acids of the target protein play a role in phosphorylation, not general binding."}','{"Correct. This control is crucial to determine if the observed phosphorylation is solely due to the enzyme''s activity and not due to any other factors, such as auto-phosphorylation or contaminants present in the buffer or modified proteins. By excluding the enzyme, the researcher can establish a baseline level of phosphorylation for each modified protein, ensuring that any increase in phosphorylation upon enzyme addition is directly attributable to the enzyme''s specific activity on that substrate. This control accounts for background noise or non-enzyme-mediated phosphorylation, which is essential for accurate interpretation of the results."}','{"Incorrect. Determining the optimal pH is important for characterizing the enzyme''s activity in general, but it doesn''t directly control for the specificity of the enzyme towards different modified protein substrates. While pH can influence enzyme activity, it doesn''t isolate the effect of amino acid substitutions on the phosphorylation rate, which is the primary focus of the experiment. This option addresses optimization of enzyme conditions, rather than directly controlling for the effects of substrate modifications on enzyme activity."}',3,2,'A researcher is studying a novel enzyme that catalyzes the phosphorylation of a specific protein in a signaling pathway. To investigate the enzyme''s substrate specificity, the researcher designs an experiment where they incubate the enzyme with several modified versions of the target protein. Each modified protein has a single amino acid substitution at a different location within the protein sequence. The researcher measures the rate of phosphorylation for each modified protein and compares it to the rate observed with the wild-type protein. Which of the following is the MOST significant control the researcher should include in this experiment to ensure the validity of their conclusions regarding substrate specificity?');
INSERT INTO Users_question VALUES(19,'{''1. The mutation disrupted hydrophobic interactions necessary for substrate binding.''}','{''2. The mutation altered the overall secondary structure of the enzyme, leading to misfolding.''}','{''3. The mutation eliminated an ionic interaction between the enzyme and the protein substrate.''}','{"4. The mutation increased the enzyme''s affinity for a competitive inhibitor."}','{''Incorrect. While alanine is hydrophobic, the original glutamic acid was charged and thus likely involved in ionic or hydrogen bonding, not hydrophobic interactions. This option presents a common pitfall: focusing solely on the introduced amino acid (alanine) and overlooking the importance of the original amino acid (glutamic acid) and the electrostatic interactions it likely formed. Students might incorrectly assume that any nonpolar amino acid is involved in hydrophobic interactions.''}','{"Incorrect. While it''s *possible* that changing a single amino acid could disrupt the overall secondary structure, it is less likely than disruption of a key interaction at the active site. Mutations that drastically alter protein folding usually involve more significant changes or occur in regions critical for maintaining the protein''s overall shape. This option is a distracter that introduces a more complex scenario (protein misfolding), which is less likely than a direct effect on substrate binding or catalysis. While the possibility exists, other options more directly address the impact on the active site given the information provided."}','{''Correct. Glutamic acid is negatively charged and is likely forming an ionic bond or a salt bridge with a positively charged amino acid on the protein substrate or a cofactor. Replacing glutamic acid with alanine, which has a nonpolar side chain, eliminates this ionic interaction, weakening the enzyme-substrate complex and decreasing the phosphorylation activity. This explanation directly addresses the change in charge and the effect on substrate binding, representing the most plausible explanation for reduced activity. A strong understanding of amino acid properties is necessary to identify this answer.''}','{"Incorrect. There''s no information provided that suggests the presence of a competitive inhibitor. This is an extraneous piece of information. Even if a competitive inhibitor were present, replacing a negatively charged residue with a nonpolar one is unlikely to specifically *increase* affinity for the inhibitor. This option serves as a distracter, introducing an irrelevant concept (competitive inhibition) that might confuse students who are not carefully analyzing the specific details given in the question stem. It requires the student to realize the lack of evidence for any change of inhibitor binding."}',3,2,'A researcher is studying a novel enzyme that catalyzes the phosphorylation of a specific protein within a cell. Site-directed mutagenesis is used to replace a single amino acid within the enzyme''s active site. Specifically, a glutamic acid residue (negatively charged) is replaced with an alanine residue (nonpolar, hydrophobic). After this mutation, the enzyme''s activity is significantly reduced. Which of the following is the MOST likely explanation for the reduced enzyme activity?');
INSERT INTO Users_question VALUES(20,'{''1. The mutation disrupted hydrophobic interactions necessary for substrate binding.''}','{''2. The mutation altered the overall secondary structure of the enzyme, leading to misfolding.''}','{''3. The mutation eliminated an ionic interaction between the enzyme and the protein substrate.''}','{"4. The mutation increased the enzyme''s affinity for a competitive inhibitor."}','{''Incorrect. While alanine is hydrophobic, the original glutamic acid was charged and thus likely involved in ionic or hydrogen bonding, not hydrophobic interactions. This option presents a common pitfall: focusing solely on the introduced amino acid (alanine) and overlooking the importance of the original amino acid (glutamic acid) and the electrostatic interactions it likely formed. Students might incorrectly assume that any nonpolar amino acid is involved in hydrophobic interactions.''}','{"Incorrect. While it''s *possible* that changing a single amino acid could disrupt the overall secondary structure, it is less likely than disruption of a key interaction at the active site. Mutations that drastically alter protein folding usually involve more significant changes or occur in regions critical for maintaining the protein''s overall shape. This option is a distracter that introduces a more complex scenario (protein misfolding), which is less likely than a direct effect on substrate binding or catalysis. While the possibility exists, other options more directly address the impact on the active site given the information provided."}','{''Correct. Glutamic acid is negatively charged and is likely forming an ionic bond or a salt bridge with a positively charged amino acid on the protein substrate or a cofactor. Replacing glutamic acid with alanine, which has a nonpolar side chain, eliminates this ionic interaction, weakening the enzyme-substrate complex and decreasing the phosphorylation activity. This explanation directly addresses the change in charge and the effect on substrate binding, representing the most plausible explanation for reduced activity. A strong understanding of amino acid properties is necessary to identify this answer.''}','{"Incorrect. There''s no information provided that suggests the presence of a competitive inhibitor. This is an extraneous piece of information. Even if a competitive inhibitor were present, replacing a negatively charged residue with a nonpolar one is unlikely to specifically *increase* affinity for the inhibitor. This option serves as a distracter, introducing an irrelevant concept (competitive inhibition) that might confuse students who are not carefully analyzing the specific details given in the question stem. It requires the student to realize the lack of evidence for any change of inhibitor binding."}',3,2,'A researcher is studying a novel enzyme that catalyzes the phosphorylation of a specific protein within a cell. Site-directed mutagenesis is used to replace a single amino acid within the enzyme''s active site. Specifically, a glutamic acid residue (negatively charged) is replaced with an alanine residue (nonpolar, hydrophobic). After this mutation, the enzyme''s activity is significantly reduced. Which of the following is the MOST likely explanation for the reduced enzyme activity?');
INSERT INTO Users_question VALUES(21,'{''1. Valine to Proline''}','{''2. Glutamate to Alanine''}','{''3. Leucine to Serine''}','{''4. Phenylalanine to Tryptophan''}','{"Incorrect. Valine is a nonpolar, hydrophobic amino acid, while proline is also hydrophobic but has a unique cyclic structure that introduces rigidity. Replacing valine with proline could disrupt the protein''s flexibility, but is unlikely to specifically prevent heat-induced aggregation. If anything, the rigidity could increase aggregation. Proline''s structure limits the conformational freedom of the polypeptide chain, which is not the issue here as the aggregation is happening due to the affect of temperature."}','{''Incorrect. Glutamate is a negatively charged (acidic), polar amino acid, while alanine is a nonpolar, hydrophobic amino acid. Replacing glutamate with alanine would eliminate a negative charge and introduce a hydrophobic interaction. This change is more likely to *promote* aggregation at high temperatures because it removes a potential site for hydrogen bonding with water. The loss of a polar residue may cause increased hydrophobic interactions, leading to aggregation.''}','{"Correct. Leucine is a nonpolar, hydrophobic amino acid, while serine is a polar amino acid with a hydroxyl group that can form hydrogen bonds. Replacing leucine with serine introduces a polar residue that can interact with water molecules, increasing the protein''s solubility and reducing the likelihood of hydrophobic aggregation at high temperatures. Hydrophobic interactions are a major driving force in protein aggregation, so introducing a polar amino acid can disrupt these interactions and prevent aggregation."}','{"Incorrect. Both phenylalanine and tryptophan are aromatic, hydrophobic amino acids. While tryptophan is larger than phenylalanine, the change is not significant enough to substantially change its hydrophobic properties in a way that would prevent aggregation. Both amino acids prefer to interact with other hydrophobic residues, so this substitution would likely have little to no effect on the protein''s stability at high temperatures. Aromatic amino acids often contribute to hydrophobic packing within proteins, and swapping one for another is unlikely to disrupt the overall hydrophobic interactions that cause aggregation."}',3,2,'A researcher is studying a novel protein and observes that it aggregates and becomes non-functional at high temperatures. Site-directed mutagenesis is used to replace a single amino acid within the protein, and the resulting mutant protein no longer aggregates at high temperatures. Which of the following amino acid substitutions is MOST likely to have restored the protein''s stability?');
INSERT INTO Users_question VALUES(22,'{''1. The mutation decreases the affinity of the enzyme for its substrate.''}','{''2. The mutation decreases the maximum velocity of the enzyme-catalyzed reaction.''}','{''3. The mutation increases the catalytic efficiency of the enzyme.''}','{''4. The mutation affects the turnover number of the enzyme.''}','{"Correct. The Michaelis constant (*K*M) is a measure of the affinity of an enzyme for its substrate. A higher *K*M value indicates lower affinity. The MUT ALT has a *K*M of 20 mM, which is significantly higher than the WT ALT''s *K*M of 5 mM. Therefore, the mutation decreases the enzyme''s affinity for its substrate. Students may incorrectly focus on the Vmax, which is unchanged, leading them to disregard changes in affinity."}','{''Incorrect. The maximum velocity (*V*max) represents the rate of the reaction when the enzyme is saturated with substrate. According to the data, the *V*max of the MUT ALT is equal to that of WT ALT (10 μmol/min). Therefore, the mutation does *not* decrease the *V*max. Choosing this answer suggests misunderstanding of *V*max and its representation of the maximum rate under saturated conditions. Students often confuse changes in Km with Vmax.''}','{''Incorrect. Catalytic efficiency is often represented by the ratio *V*max/*K*M. While the *V*max is unchanged, the *K*M has increased in the mutant. Therefore, the catalytic efficiency (Vmax/Km) decreases. *V*max/*K*M for WT ALT is (10 μmol/min) / (5 mM) = 2. *V*max/*K*M for MUT ALT is (10 μmol/min) / (20 mM) = 0.5. Therefore, the mutation actually *decreases* catalytic efficiency. This option is tempting because students might assume that any alteration to an enzyme improves its catalytic properties.''}','{''Incorrect. The turnover number (kcat) refers to the number of substrate molecules one enzyme molecule can convert per unit time when the enzyme is fully saturated with substrate. Since the *V*max is unchanged, and *V*max is related to kcat, the turnover number is also unchanged, assuming that the enzyme concentration has not changed in the experiment. Students often confuse Km with Kcat and think that because the Km changes, so does the Kcat.''}',1,2,replace('A researcher is studying the effect of a novel mutation in the gene encoding the enzyme alanine aminotransferase (ALT) on its catalytic activity. They purify the wild-type (WT) and mutant (MUT) enzymes and measure their reaction rates at various substrate concentrations. The data is analyzed using Lineweaver-Burk plots. The researcher determines the following kinetic parameters:\n\n*   **WT ALT:** *K*M = 5 mM, *V*max = 10 μmol/min\n*   **MUT ALT:** *K*M = 20 mM, *V*max = 10 μmol/min\n\nBased on this data, what is the best conclusion about the effect of the mutation on ALT?','\n',char(10)));
INSERT INTO Users_question VALUES(23,'{''1. Glutamic acid''}','{''2. Lysine''}','{''3. Valine''}','{''4. Histidine''}','{''Glutamic acid is a negatively charged, acidic amino acid. Due to its charged nature, it is highly hydrophilic and thus unlikely to be found within the hydrophobic core of a transmembrane domain. Charged amino acids prefer to interact with the polar environment of the aqueous solution or the polar head groups of the lipid membrane. Many students forget that transmembrane proteins have a region that spans the membrane.''}','{''Lysine is a positively charged, basic amino acid. Similar to glutamic acid, its charged side chain makes it hydrophilic and unfavorable for residing in the hydrophobic transmembrane domain. Lysine would prefer to interact with the polar regions of the protein or the aqueous environment. A common mistake is to confuse polarity with being hydrophobic.''}','{"Valine is a nonpolar, hydrophobic amino acid. Its nonpolar side chain allows it to interact favorably with the hydrophobic lipid tails within the cell membrane''s interior. Therefore, it''s highly likely to be found in the transmembrane domain. Students should memorize the amino acids that are non polar. Another common trap is to select an answer that contains some polar and non polar amino acids."}','{''Histidine has an imidazole ring that can be protonated or deprotonated depending on the pH of the environment. While it can sometimes participate in hydrophobic interactions when deprotonated, its tendency to carry a charge makes it less likely to be a primary constituent of a transmembrane domain compared to a purely hydrophobic amino acid like valine. Though it can be found in the transmembrane protein at a specific pH it is not the best choice as it depends on the environmental conditions. Some students get tricked here as Histidine can have both hydrophobic and hydrophilic characteristics depending on its protonation state.''}',3,2,'Which of the following amino acids is most likely to be found in the transmembrane domain of a protein?');
INSERT INTO Users_question VALUES(24,'{''1. Glutamic acid''}','{''2. Lysine''}','{''3. Valine''}','{''4. Histidine''}','{''Glutamic acid is a negatively charged, acidic amino acid. Due to its charged nature, it is highly hydrophilic and thus unlikely to be found within the hydrophobic core of a transmembrane domain. Charged amino acids prefer to interact with the polar environment of the aqueous solution or the polar head groups of the lipid membrane. Many students forget that transmembrane proteins have a region that spans the membrane.''}','{''Lysine is a positively charged, basic amino acid. Similar to glutamic acid, its charged side chain makes it hydrophilic and unfavorable for residing in the hydrophobic transmembrane domain. Lysine would prefer to interact with the polar regions of the protein or the aqueous environment. A common mistake is to confuse polarity with being hydrophobic.''}','{"Valine is a nonpolar, hydrophobic amino acid. Its nonpolar side chain allows it to interact favorably with the hydrophobic lipid tails within the cell membrane''s interior. Therefore, it''s highly likely to be found in the transmembrane domain. Students should memorize the amino acids that are non polar. Another common trap is to select an answer that contains some polar and non polar amino acids."}','{''Histidine has an imidazole ring that can be protonated or deprotonated depending on the pH of the environment. While it can sometimes participate in hydrophobic interactions when deprotonated, its tendency to carry a charge makes it less likely to be a primary constituent of a transmembrane domain compared to a purely hydrophobic amino acid like valine. Though it can be found in the transmembrane protein at a specific pH it is not the best choice as it depends on the environmental conditions. Some students get tricked here as Histidine can have both hydrophobic and hydrophilic characteristics depending on its protonation state.''}',3,2,'Which of the following amino acids is most likely to be found in the transmembrane domain of a protein?');
INSERT INTO Users_question VALUES(25,'{''1. Increasing the enzyme concentration in both the presence and absence of the inhibitor.''}','{''2. Performing experiments at several different, known inhibitor concentrations and determining the apparent Km at each concentration.''}','{''3. Measuring the Vmax of the reaction in both the presence and absence of the inhibitor.''}','{''4. Using a non-competitive inhibitor instead of a competitive inhibitor and measuring the change in Vmax.''}','{"Incorrect. While increasing enzyme concentration will increase the reaction rate, it does not directly provide information about the inhibitor''s binding affinity (Ki). Ki is a measure of how tightly the inhibitor binds to the enzyme. Changing enzyme concentration will simply shift the reaction rate curves up or down, but it won''t change the relationship between substrate concentration and reaction rate in a way that reveals Ki. A common pitfall is to confuse changes in reaction rate with changes in binding affinity. Ki is an equilibrium constant specifically for the inhibitor-enzyme interaction, not simply how fast the reaction goes."}','{"Correct. The Ki value can be determined by analyzing how the apparent Km changes at different inhibitor concentrations. In the presence of a competitive inhibitor, the apparent Km increases (Km,app > Km), because a higher substrate concentration is needed to reach Vmax/2. By determining the Km,app at several known concentrations of the inhibitor, the Ki can be calculated using the equation: Km,app = Km(1 + [I]/Ki). A common approach involves plotting Km,app vs. [I] and determining Ki from the slope. This approach directly addresses the question of inhibitor binding and provides a quantifiable measure (Ki) of that binding. Students sometimes overlook this method because it requires multiple experiments and data analysis, but it''s a standard technique for determining Ki."}','{"Incorrect. Competitive inhibitors do not affect Vmax; they only affect Km. Measuring Vmax will simply confirm that the inhibitor is indeed competitive, but it won''t tell you anything about the strength of the inhibitor''s binding (Ki). Vmax is the maximum rate of the reaction when the enzyme is saturated with substrate, and a competitive inhibitor can be overcome by increasing substrate concentration to reach Vmax. Students often make the mistake of thinking that any change in reaction kinetics provides information about binding affinity. However, Vmax is related to the enzyme concentration and catalytic efficiency, not the inhibitor''s binding."}','{''Incorrect. While non-competitive inhibitors do affect Vmax, measuring the change in Vmax only provides information about the efficacy of the non-competitive inhibitor, not the binding affinity of the *original* competitive inhibitor which is of interest in the question prompt. The Ki for a non-competitive inhibitor can be determined from the change in Vmax, but this does not address the question of how to find the Ki of the competitive inhibitor being studied. A pitfall here is to change the type of experiment instead of modifying the existing experimental setup to answer the research question.''}',2,2,'A researcher is studying the effects of a novel competitive inhibitor on an enzyme-catalyzed reaction. They perform two sets of experiments: one with the enzyme alone and one with the enzyme and the inhibitor. In both sets, they measure the initial reaction rate at various substrate concentrations. Which of the following experimental modifications would BEST allow the researcher to determine the inhibitor''s binding affinity (Ki) to the enzyme?');
INSERT INTO Users_question VALUES(26,'{''1. Increasing the enzyme concentration in both the presence and absence of the inhibitor.''}','{''2. Performing experiments at several different, known inhibitor concentrations and determining the apparent Km at each concentration.''}','{''3. Measuring the Vmax of the reaction in both the presence and absence of the inhibitor.''}','{''4. Using a non-competitive inhibitor instead of a competitive inhibitor and measuring the change in Vmax.''}','{"Incorrect. While increasing enzyme concentration will increase the reaction rate, it does not directly provide information about the inhibitor''s binding affinity (Ki). Ki is a measure of how tightly the inhibitor binds to the enzyme. Changing enzyme concentration will simply shift the reaction rate curves up or down, but it won''t change the relationship between substrate concentration and reaction rate in a way that reveals Ki. A common pitfall is to confuse changes in reaction rate with changes in binding affinity. Ki is an equilibrium constant specifically for the inhibitor-enzyme interaction, not simply how fast the reaction goes."}','{"Correct. The Ki value can be determined by analyzing how the apparent Km changes at different inhibitor concentrations. In the presence of a competitive inhibitor, the apparent Km increases (Km,app > Km), because a higher substrate concentration is needed to reach Vmax/2. By determining the Km,app at several known concentrations of the inhibitor, the Ki can be calculated using the equation: Km,app = Km(1 + [I]/Ki). A common approach involves plotting Km,app vs. [I] and determining Ki from the slope. This approach directly addresses the question of inhibitor binding and provides a quantifiable measure (Ki) of that binding. Students sometimes overlook this method because it requires multiple experiments and data analysis, but it''s a standard technique for determining Ki."}','{"Incorrect. Competitive inhibitors do not affect Vmax; they only affect Km. Measuring Vmax will simply confirm that the inhibitor is indeed competitive, but it won''t tell you anything about the strength of the inhibitor''s binding (Ki). Vmax is the maximum rate of the reaction when the enzyme is saturated with substrate, and a competitive inhibitor can be overcome by increasing substrate concentration to reach Vmax. Students often make the mistake of thinking that any change in reaction kinetics provides information about binding affinity. However, Vmax is related to the enzyme concentration and catalytic efficiency, not the inhibitor''s binding."}','{''Incorrect. While non-competitive inhibitors do affect Vmax, measuring the change in Vmax only provides information about the efficacy of the non-competitive inhibitor, not the binding affinity of the *original* competitive inhibitor which is of interest in the question prompt. The Ki for a non-competitive inhibitor can be determined from the change in Vmax, but this does not address the question of how to find the Ki of the competitive inhibitor being studied. A pitfall here is to change the type of experiment instead of modifying the existing experimental setup to answer the research question.''}',2,2,'A researcher is studying the effects of a novel competitive inhibitor on an enzyme-catalyzed reaction. They perform two sets of experiments: one with the enzyme alone and one with the enzyme and the inhibitor. In both sets, they measure the initial reaction rate at various substrate concentrations. Which of the following experimental modifications would BEST allow the researcher to determine the inhibitor''s binding affinity (Ki) to the enzyme?');
CREATE TABLE IF NOT EXISTS "Users_user_question" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "CORRECTLY_ANSWERED" bool NOT NULL, "QUESTION_ID_id" integer NOT NULL REFERENCES "Users_question" ("QUESTION_ID") DEFERRABLE INITIALLY DEFERRED, "TOPIC_id" integer NOT NULL REFERENCES "Users_topic" ("TOPIC_ID") DEFERRABLE INITIALLY DEFERRED, "USER_ID_id" integer NOT NULL REFERENCES "Users_user" ("USER_ID") DEFERRABLE INITIALLY DEFERRED, "ANSWERED_ON" datetime NULL);
INSERT INTO Users_user_question VALUES(18,0,3,2,1,'2025-02-12 23:46:41.750213');
INSERT INTO Users_user_question VALUES(19,0,5,2,1,'2025-02-12 23:46:41.759066');
INSERT INTO Users_user_question VALUES(20,0,6,2,1,'2025-02-12 23:46:41.765010');
INSERT INTO Users_user_question VALUES(21,0,7,2,1,'2025-02-12 23:46:41.770670');
INSERT INTO Users_user_question VALUES(22,0,8,2,1,'2025-02-12 23:59:47.395389');
INSERT INTO Users_user_question VALUES(23,0,9,2,1,'2025-02-13 00:01:32.990199');
INSERT INTO Users_user_question VALUES(24,0,10,2,1,'2025-02-13 00:01:52.545630');
INSERT INTO Users_user_question VALUES(25,0,11,2,1,'2025-02-13 00:02:57.683888');
INSERT INTO Users_user_question VALUES(26,0,12,2,1,'2025-02-13 00:04:54.584080');
INSERT INTO Users_user_question VALUES(27,0,13,2,1,'2025-02-13 00:07:08.261427');
INSERT INTO Users_user_question VALUES(28,0,14,2,1,'2025-02-13 00:07:15.599280');
INSERT INTO Users_user_question VALUES(29,0,15,2,1,'2025-02-13 00:09:09.787137');
INSERT INTO Users_user_question VALUES(30,0,16,2,1,'2025-02-13 00:09:54.749958');
INSERT INTO Users_user_question VALUES(31,0,17,2,1,'2025-02-13 00:11:54.513236');
INSERT INTO Users_user_question VALUES(32,0,18,2,1,'2025-02-13 00:12:01.785258');
INSERT INTO Users_user_question VALUES(33,0,19,2,1,'2025-02-13 00:14:00.774094');
INSERT INTO Users_user_question VALUES(34,0,20,2,1,'2025-02-13 00:14:09.238645');
INSERT INTO Users_user_question VALUES(35,0,21,2,1,'2025-02-13 00:15:05.465933');
INSERT INTO Users_user_question VALUES(36,0,22,2,1,'2025-02-13 00:16:52.781968');
INSERT INTO Users_user_question VALUES(37,0,23,2,1,'2025-02-13 00:17:28.949092');
INSERT INTO Users_user_question VALUES(38,0,24,2,1,'2025-02-13 00:17:33.002131');
INSERT INTO Users_user_question VALUES(39,0,25,2,1,'2025-02-13 00:17:44.895656');
INSERT INTO Users_user_question VALUES(40,0,26,2,1,'2025-02-13 00:18:10.006561');
DELETE FROM sqlite_sequence;
INSERT INTO sqlite_sequence VALUES('django_migrations',22);
INSERT INTO sqlite_sequence VALUES('django_admin_log',27);
INSERT INTO sqlite_sequence VALUES('django_content_type',10);
INSERT INTO sqlite_sequence VALUES('auth_permission',40);
INSERT INTO sqlite_sequence VALUES('auth_group',0);
INSERT INTO sqlite_sequence VALUES('auth_user',1);
INSERT INTO sqlite_sequence VALUES('Users_user_question',40);
CREATE UNIQUE INDEX "auth_group_permissions_group_id_permission_id_0cd325b0_uniq" ON "auth_group_permissions" ("group_id", "permission_id");
CREATE INDEX "auth_group_permissions_group_id_b120cbf9" ON "auth_group_permissions" ("group_id");
CREATE INDEX "auth_group_permissions_permission_id_84c5c92e" ON "auth_group_permissions" ("permission_id");
CREATE UNIQUE INDEX "auth_user_groups_user_id_group_id_94350c0c_uniq" ON "auth_user_groups" ("user_id", "group_id");
CREATE INDEX "auth_user_groups_user_id_6a12ed8b" ON "auth_user_groups" ("user_id");
CREATE INDEX "auth_user_groups_group_id_97559544" ON "auth_user_groups" ("group_id");
CREATE UNIQUE INDEX "auth_user_user_permissions_user_id_permission_id_14a6b632_uniq" ON "auth_user_user_permissions" ("user_id", "permission_id");
CREATE INDEX "auth_user_user_permissions_user_id_a95ead1b" ON "auth_user_user_permissions" ("user_id");
CREATE INDEX "auth_user_user_permissions_permission_id_1fbb5f2c" ON "auth_user_user_permissions" ("permission_id");
CREATE INDEX "django_admin_log_content_type_id_c4bce8eb" ON "django_admin_log" ("content_type_id");
CREATE INDEX "django_admin_log_user_id_c564eba6" ON "django_admin_log" ("user_id");
CREATE UNIQUE INDEX "django_content_type_app_label_model_76bd3d3b_uniq" ON "django_content_type" ("app_label", "model");
CREATE UNIQUE INDEX "auth_permission_content_type_id_codename_01ab375a_uniq" ON "auth_permission" ("content_type_id", "codename");
CREATE INDEX "auth_permission_content_type_id_2f476e4b" ON "auth_permission" ("content_type_id");
CREATE INDEX "django_session_expire_date_a5c62663" ON "django_session" ("expire_date");
CREATE INDEX "Users_user_TOPIC_id_e37c28d0" ON "Users_user" ("TOPIC_id");
CREATE INDEX "Users_question_TOPIC_id_e3d1d416" ON "Users_question" ("TOPIC_id");
CREATE INDEX "Users_user_question_QUESTION_ID_id_ecbbfc5a" ON "Users_user_question" ("QUESTION_ID_id");
CREATE INDEX "Users_user_question_TOPIC_id_6c60a1de" ON "Users_user_question" ("TOPIC_id");
CREATE INDEX "Users_user_question_USER_ID_id_b2b6c4e9" ON "Users_user_question" ("USER_ID_id");
CREATE UNIQUE INDEX "Users_topic_TOPIC_NAME_SUBTOPIC_NAME_SUBJECT_e9ef5c26_uniq" ON "Users_topic" ("TOPIC_NAME", "SUBTOPIC_NAME", "SUBJECT");
CREATE UNIQUE INDEX "Users_user_question_USER_ID_id_QUESTION_ID_id_61490c08_uniq" ON "Users_user_question" ("USER_ID_id", "QUESTION_ID_id");
COMMIT;
